android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -371,6 +366,24 @@ public void setProgress(int max, int progress, boolean indeterminate)
 		notificationBuilder.setProgress(max, progress, indeterminate);
 	}
 
+	@Kroll.method
+	public void addAction(Object icon, String title, PendingIntentProxy pendingIntent)
+	{
+		int iconId = -1;
+		if (icon instanceof Number) {
+			iconId = ((Number)icon).intValue();
+		} else {
+			String iconUrl = TiConvert.toString(icon);
+			if (iconUrl == null) {
+				Log.e(TAG, "Url is null");
+				return;
+			}
+			String iconFullUrl = resolveUrl(null, iconUrl);
+			iconId = TiUIHelper.getResourceId(iconFullUrl);
+		}
+		notificationBuilder.addAction(iconId, title, pendingIntent.getPendingIntent());
```

Comment:
And if "pendingIntent" is null, we should log an error or throw an exception indicating that this argument cannot be null.